docs/modules/ROOT/pages/capacity-planning.adoc

```diff
@@ -180,17 +187,18 @@ times.
 NOTE: If you are an Enterprise customer using the High-Density Memory Store
 with large data sizes, we recommend a large increase in partition count, starting with 5009 or higher.
 
-You cannot change the partition count after a cluster is created,
-so if you have a larger cluster, be sure to test and
-set an optimum partition count prior to deployment.
-If you need to change the partition count after a cluster is running,
-you will need to schedule a maintenance window to update the partition count
-and restart the cluster.
+NOTE: The partition count cannot be eaily changed after a cluster is created, so if you have a large cluster 
+be sure to test and set an optimum partition count prior to deployment. If you need to change th partition 
+count after a cluster is already running, you will need to schedule a maintenance window to entirely bring 
+the cluster down. If your cluster uses xref:storage:persistence.adoc[Persistence] or xref:cp-subsystem:persistence.adoc[CP Persistence] 
+features, those persistent files will need to be removed after the cluster is shut down, as they contain 
```

Comment:
... If your cluster uses the xref:storage:persistence.adoc[Persistence] or xref:cp-subsystem:persistence.adoc[CP Persistence] feature, those ...

Reply:
I've just realized we use the same information given in this paragraph also in the Production Checklist section. Note for docs team.

Reply:
Yes, you're correct that the information appears in both places. Personally I think it does belong in both locations, but if you see a good chance for consolidation that's fine too. (Also, changes made.)